Remotion/Development/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBase.cs

```diff
@@ -194,9 +194,9 @@ public void AssertTextNode (XmlNode parent, string content, int index)
     /// <summary>
     /// Asserts that <paramref name="node"/> has an attribute named <paramref name="attributeName"/> with value <paramref name="attributeValue"/>.
     /// </summary>
-    public void AssertAttribute (XmlNode node, string attributeName, WebString attributeValue)
+    public void AssertAttribute (XmlNode node, string attributeName, PlainTextString? attributeValue)
     {
-      AssertAttribute(node, attributeName, attributeValue.ToString(WebStringEncoding.Attribute));
+      AssertAttribute(node, attributeName, attributeValue?.GetValue());
```

Comment:
Just for clarification: the tests for null couldn't be done without much effort, correct?